urlfix/urlfix.py

```diff
@@ -43,9 +44,14 @@ def replace_urls(self, verbose=False, correct_urls=None):
         number_moved = 0
         number_of_urls = 0
 
-        if self.output_file is None:
+        if not self.output_file and not inplace:
             raise ValueError("Please provide an output file to write to.")
         output_file = self.output_file
+        if inplace:
```

Comment:
Thanks, I was looking at this last night and thought of the following:

* Would it be better to create a temporary file with `tempfile.TemporaryFile()` or `tempfile.NamedTemporaryFile()`?
* What do you think of open the file as 
```python
with open(file, "x")
```
which will check file existence and opens it for writing. The idea is to avoid creating files manually.

Reply:
I meant to review lines 50-54. For some reason only line 50 is shown, sorry!

Reply:
@Nelson-Gon I am working on this and will get back to you soon.